src/topp/FileConverter.cpp

```diff
@@ -446,7 +446,7 @@ class TOPPFileConverter :
                                                  CONVERSION_MZXML));
       MzXMLFile f;
       f.setLogType(log_type_);
-      ChromatogramTools().convertChromatogramsToSpectra<MSExperimentType>(exp);
+      //ChromatogramTools().convertChromatogramsToSpectra<MSExperimentType>(exp);
```

Comment:
hmm this is not so much legacy but (if I remember correctly) still what proteowizard does with SRM/MRM data and somehow - for better or worse - the "standard" way of doing it. 

>  It (rightfully) confuses MaxQuant and I cannot see a good reason why we should do this.

Do you see any other way to store SRM data in MzXML?